locale/ca_ES/default.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../dtd/locale.dtd">
+
+<!--
+  * locale/ca_ES/default.xml
+  *
+  * Copyright (c) 2014-2015 Simon Fraser University Library
```

Comment:
I already did batch changes for the branches we're likely to release. I suspect we won't release anything further from the omp-dev-1_1 branch, and yes, I'll correct the dates there before releasing if we do.